component/client/backgroundContext.tsx

```diff
@@ -1,7 +1,92 @@
 const backgroundContext: any = chrome.extension.getBackgroundPage();
 
-export const storage = backgroundContext.backgroundStorageService;
-export const room = backgroundContext.room;
+export interface IRoomService {
```

Comment:
I will leave it here for now. If you want to extract it, we can do that in a separate PR.